Dockerfile

```diff
@@ -1,23 +1,20 @@
-FROM alpine:3.3
+FROM alpine:3.4
 
 RUN echo "@testing http://dl-4.alpinelinux.org/alpine/edge/testing" | tee -a /etc/apk/repositories
+RUN echo "http://dl-5.alpinelinux.org/alpine/edge/main/" | tee -a /etc/apk/repositories
```

Comment:
edge/main was needed for python2 in the alpine:3.4 image, I believe. And edge/testing is for mono.